### ISSUE TITLE
Inaccessibility of Scroll Function in FlutterFlow Editor for Custom Extension Users

### ISSUE THREAD

**Issue by yoosefashraff** [FlutterFlow/flutterflow-issues#2698]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I've created a Chrome extension that allows me to scroll using an on-screen keyboard due to my condition, muscular dystrophy. This extension helps me navigate web pages using my mobile device as an input method. While the extension works well on most websites, it fails to interact with the scrolling functions within the FlutterFlow app editor. This prevents me from viewing the entire canvas while building an app, as I cannot scroll down to see the full screen.

### Expected Behavior

The expected behavior is that my scrolling extension should interact with FlutterFlow's editor, allowing me to scroll up and down within the editor’s canvas without issue.

### Steps to Reproduce

1. Open the FlutterFlow app editor.
2. Attempt to scroll within the editor using my custom Chrome extension that simulates scroll events.
3. Notice that the scrolling does not occur within the FlutterFlow editor's canvas, despite the extension working on other web pages.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITEehfLq5ZNJn7xF1pDQde0xrwMwJ0QlUIInj+0bWRMhNb3mPLQufMjCQEhLYte9YFdYD1aLuE0y3PvHiuLLNcYnHxyrbL5Z+JRXQRXLQnKgbKnREZeePWpCTO1jfna5556nuBNDKu12dGwB6lz7CvCNG3qCf9qOYwx5e6fDbOY=

### Context

I rely on my custom extension to navigate and work independently due to my muscular dystrophy. Not being able to scroll within the FlutterFlow editor significantly hampers my ability to use the platform for app development. I am seeking a solution to make the editor compatible with my accessibility tools.

### Visual documentation

https://github.com/FlutterFlow/flutterflow-issues/assets/153321798/f179e0f9-16df-4fae-9a80-4fc862605b0b

### Additional Info

I can provide the source code of my extension for your technical team's review. I have used AI technologies to develop it and am confident that minor adjustments might resolve this issue.

### Environment

```markdown
- FlutterFlow version: Can't find it but it's updated
- Platform:
- Browser name and version:Chrome 123.0.6312.123 (Official Build) (64-bit)
- Operating system and version affected: Windows 11 pro
```

**Comment by leighajarett** [comment]
Just wanted to chime in and thank you for flagging this @yoosefashraff! Love that you made your own extension to help with scrolling. I don't want you to think that us closing this issue means it isn't important to us, we just track feature requests in a different spot :) 

Is there a way we can contact you in case we have more questions / need some access to the extension?

**Comment by yoosefashraff** [comment]
Yes, I Understand this and appreciate it, yeah you can contact me whenever
you want and I can provide you with the extension code if you need it, As I
know that it's not hard to achieve this I guess it's just something with
recognizing events in between the extension and FlutterFlow But as I
mentioned I'm not a coder [programmer] I just have some of a background.
But I really need this accessibility.

Thank you
Youssef

**Comment by takashiff** [comment]
Hi @yoosefashraff, if you don't mind, could you reach out to me at takashi@flutterflow.io and send me the extension code? This would help a lot in finding what's causing this issue. Thanks

**Comment by yoosefashraff** [comment]
Yes, I will send it to you.

**Comment by takashiff** [comment]
@yoosefashraff Great, thanks so much

**Comment by hack-r** [comment]
I wonder if this is the same bug I'm experiencing, wherein you can't scroll down in the editor view to add or edit content beyond what's visible without scrolling, even though the parent column is allegedly scrollable? I'm unclear because I understand OP was discussing it in the context of a self-authored Chrome plugin, whereas mine is just the general case of using FlutterFlow.

I see this one was discussed as being a feature rather than a bug. I could be wrong, but seems like a really critical bug to me, how else can you develop pages longer than one screen? Was it ever fixed?
